content/tutorials/getting-started/nlb-environment.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "sensenet in a Network Load Balanced environment"
+metaTitle: "sensenet Tutorials - Network Load Balanced environment"
+metaDescription: "sensenet Tutorials - Configure sensenet in a Network Load Balanced environment"
+---
+
+If you are using sensenet in an [advanced app architecture](/tutorials/getting-started/02-advanced-application-architecture) with multiple repository services, you need to configure the different components a bit differently to work together. This article describes how to configure the components of a sensenet application in a network load balanced environment.
+
+## Components
+You'll need to configure the following components:
+- **IdentityServer** (authentication service, single instance)
+- **Search service** (single instance)
+- **sensenet application** (repository service, multiple instances)
+- **RabbitMQ** service for messaging
+
+> Configuring the MS SQL Server database is not covered in this article.
+
+## Communication
+For the sensenet **repository instances** to work together, they need to communicate with each other. We recommend using **RabbitMQ** for this purpose. The repository instances use RabbitMQ to send messages to each other. These messages are used to synchronize the cache between the instances. There are two kinds of connections between the repository instances:
+
+- cache invalidation and other system messages
+- security store messages
+
+These two types of messages are sent on different RabbitMQ exchanges that are configured separately. See the following sections for details.
+
+In case of using a **Search Service** there is also a **gRPC connection** from the repository instances to the Search Service. This connection is used to send indexing and search requests to the Search Service.
```

Comment:
Instead of "in case of", "because" would be better, and I would add gRPC to the components. See ChannelOptions in config:
``` json
{
  "sensenet": {
    "search": {
      "service": {
        "ServiceAddress": "value",
        "ValidateServerCertificate": "value",
        "ChannelOptions": { ... }
```